test/image/mocks/line_scatter.json

```diff
@@ -55,6 +55,13 @@
       "marker": {
         "opacity": 0.707
       }
+    },
+    {
+      "x": [1, 2, 3],
+      "y": [1, 2, 3],
+      "opacity": 0.2,
+      "line": { "width": 10, "color": "red"},
+      "marker": { "size": 20, "color": "blue"}
```

Comment:
![image](https://cloud.githubusercontent.com/assets/6675409/21866540/9ce98476-d818-11e6-8eb5-a833fabd32e3.png)

looks like it is done _right_.